python/mlc_chat/op/moe.py

```diff
@@ -0,0 +1,776 @@
+"""Mixture of Experts operators"""
+import tvm
+from tvm import DataType, te, tir
+from tvm.relax.frontend.nn import Tensor, op
+from tvm.script import tir as T
+from tvm.topi.cuda.sort import topk as topi_topk
+
+from mlc_chat.support import logging
+
+logger = logging.getLogger(__name__)
+
+# pylint: skip-file
+# mypy: ignore-errors
```

Comment:
Usually we don't disable all possible error codes in the entire file, because linters are intentionally designed to ensure code stype/quality. Indeed there could be some false positives, e.g. mypy cannot find certain attributes in TVMScript. An example of skipping: https://github.com/mlc-ai/mlc-llm/blob/095858cc6f9b01b5a02f1e8b3dd4510903ced31e/python/mlc_chat/op/position_embedding.py#L45